src/opnsense/mvc/app/views/OPNsense/Core/hasync_status.volt

```diff
@@ -110,17 +110,25 @@
                 $("#status_error").show();
             }
             $("#status_query").hide();
-        });
-        $("#act_restart_all").click(function(){
-            let icon = $(this).find('i');
-            if (icon.hasClass('spinner')) {
-                return;
-            }
-            icon.removeClass('fa-repeat').addClass('fa-spinner fa-pulse');
-            ajaxCall('/api/core/hasync_status/restart_all', {}, function(data){
-                icon.removeClass('fa-spinner fa-pulse').addClass('fa-repeat');
-                $('#grid_services').bootgrid('reload');
-            });
+
+            $("#grid_services-header > .row > .actionBar").prepend($(`
+                <div class="pull-left" style="position: absolute; top: 50%; transform:translateY(-50%);">
+                    <span> {{ lang._('Synchronize and reconfigure all') }} </span>
+                    <span id="act_restart_all" class="btn btn-xs btn-default" data-toggle="tooltip" title="{{ lang._('Synchronize and restart all services') }}">
+                        <i class="fa fa-repeat fa-fw"></i>
+                    </span>
+                </div>
+            `).click(function() {
```

Comment:
doesn't this hook the `click()` event to `.actionBar` instead of the button?

Reply:
the click is called on the inner jquery object, no the actionbar selector, the formatting is a bit confusing